### PR TITLE
refactor(components): [time-picker] use useNamespace

### PR DESCRIPTION
**Pull request by gjfei**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by sxzz** [comment]
@JeremyWuuuuu Could you please review scss?

File: packages/components/time-picker/src/common/picker.vue [hunk 1]
```diff
@@ -44,7 +43,7 @@
         <template #prefix>
           <el-icon
             v-if="triggerIcon"
-            class="el-input__icon"
+            :class="`${ns.namespace.value}-input__icon`"
```
Comment (sxzz) [inline packages/components/time-picker/src/common/picker.vue:46]: Please add `nsInput`

**Review by sxzz — COMMENTED** [review]
Please add multi namespace

**Comment by JeremyWuuuuu** [comment]
LGTM on style side. 🏓  /cc @sxzz

**Review by sxzz — APPROVED** [review]

**Pull request merged**